lts_array/ltsva.py

```diff
@@ -96,14 +96,20 @@ def ltsva(st, rij, winlen, winover, alpha=1.0):
         # Return data structure filled with NaNs if true.
         dataspike = np.all(tdelay == 0)
         if dataspike:
-            raise Exception("Tdelays are equal. LTS algorithm not run. \
+            print("Tdelays are equal. LTS algorithm not run. \
```

Comment:
Perhaps this is more clear?
```suggestion
            print("Time delays are equal. LTS algorithm not run. \
```